news/2020-03-18-new-featured-artist-leaf.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "New Featured Artist: LeaF"
+date: 2020-03-18 06:00:00 +0000
+---
+
+We're proud to welcome **LeaF** aboard as our latest addition to the Featured Artist roster!
+
+![](https://assets.ppy.sh/artists/73/header.jpg)
+
+Those with particularly calamitous fortunes are likely well acquainted with the wonder that is **LeaF** and his genre-defying rhythmic mastery. From tracks like *Calamity Fortune*, *MEPHISTO*, *Wizdomiot*, *I*, *Evanescent* and more, we've seen a LOT of **LeaF** over the course of osu!'s history, with many of the beatmaps charted to his work serving as cultural icons in their own right.
+
+We've lined up **30** tracks for you to test your mapping mettle with, in a variety of short and extended versions to boot.
+
+In case you're not in the know, check out a sample of some of his work below:
+
+### Aleph-0
```

Comment:
there are popular maps for this one

https://osu.ppy.sh/beatmapsets/791900
https://osu.ppy.sh/beatmapsets/518951
https://osu.ppy.sh/beatmapsets/884669

Reply:
rip pikaquim

Reply:
graveyard is the main issue as well